lms/djangoapps/edxnotes/decorators.py

```diff
@@ -45,7 +45,7 @@ def get_html(self, *args, **kwargs):
                     "courseId": unicode(self.runtime.course_id).encode("utf-8"),
                     "token": get_id_token(self.runtime.get_real_user(self.runtime.anonymous_student_id)),
                     "tokenUrl": get_token_url(self.runtime.course_id),
-                    "endpoint": get_endpoint(),
+                    "endpoint": get_public_endpoint(),
```

Comment:
Based on my code dive I think this is the place where we provide a public facing edx-notes-api endpoint to the HTML module client.